dirty_cat/_fuzzy_join.py

```diff
@@ -200,10 +199,12 @@ def fuzzy_join(left_table: pd.DataFrame,
 
     lt_arr = np.array(lt)
     rt_arr = np.array(rt)
```

Comment:
Might be nitpicking, but I still think that code should be fully human-readable. We're not in the Unix days anymore, where abbreviations were used all over the place due to space limitations :p
For example here I think it should read:
```suggestion
    left_array = np.array(left)
    right_array = np.array(right)
```
(though it might also be a good idea to rename "lt" / "left" to something more descriptive, like "left_table", or "left_table_clean")